react/week4/github-user-search/src/FetchUsers.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect, createContext } from 'react';
+import SearchInput from './SearchInput';
+
+export const DataContext = createContext('');
+
+function FetchUsers() {
+    const [error, setError] = useState(null);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [users, setUsers] = useState([]);
+
+    const fetchGithubApi = async (query) => {
+        const API_URL = `https://api.github.com/search/users?q=${query}`;
+
+        try {
+            const response = await fetch(API_URL);
+            if (!response.ok) {
+                const message = `An error has occured: ${response.status}`;
+                throw Error(message);
+            }
+            const data = await response.json();
+            setIsLoaded(true);
```

Comment:
This is set to true with each fetch but never set to false (only the initial state is false). Loading should be shown with each keystroke as it is shown in on the gif here:
https://github.com/HackYourFuture-CPH/React/blob/master/week4/homework.md

Reply:
Thanks for the feedback..
i tried to implement the feedbacks you have given.